<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductView</class>
 <widget class="QMainWindow" name="ProductView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="product_image">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>350</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Files/Images/ax man by nightrhino.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="favorite_button">
        <property name="text">
         <string>Favorite</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Files/Icons/star-2-16.ico</normaloff>
          <normalon>../Files/Icons/star-2-16 (1).ico</normalon>
          <activeoff>../Files/Icons/star-2-16 (1).ico</activeoff>../Files/Icons/star-2-16.ico</iconset>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="sites_area">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="sites_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>348</width>
           <height>196</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>title</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>specifications</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="specifications_area">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="specifications_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>177</width>
           <height>260</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
